### PR TITLE
FBISCC-69: update designs for representative-details page

### PR DESCRIPTION
**Pull request by MaxOSullivan**
### What
Updates '/representative-details' page to be in line with v5 designs.
### Why
To match designs that have been validated in research with users.
### How
Update field labels to match new desired inputs. Add validation to new fields. Add conditional field labels for inside/outside UK routes.
### Test
Added content.spec and validation.spec to 05 - representative details folder
### Screenshot
Screenshot 2020-12-01 at 12 33 28

### CODE REVIEW

File: assets/scss/custom.scss [hunk 1]
```diff
@@ -92,7 +93,7 @@ input[type="submit"].link {
     width: 100%;
   }
 
-  #representative-name, #representative-phone {
+  #representative-first-names, #representative-last-names {
```
Comment (AlexFranklinHODDaT) [inline assets/scss/custom.scss:96]: These should be 100%, move into the above CSS styling group

File: test/ui/v2/05 - representative details/content.spec.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
+'use strict';
```
Comment (AlexFranklinHODDaT) [inline test/ui/v2/05 - representative details/content.spec.js:1]: Hyphenate test directory name

File: test/ui/v2/05 - representative details/validation.spec.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+          const errorSummaries = await getErrorSummaries();
+          const errorMessages = await getErrorMessages();
+
+          expect(errorSummaries.length).to.equal(1);
+          expect(errorMessages.length).to.equal(1);
+          expect(await errorSummaries[0].innerText()).to.equal('Enter your last names');
+          expect(await errorMessages[0].innerText()).to.equal('Enter your last names');
+        });
+
```
Comment (AlexFranklinHODDaT) [inline test/ui/v2/05 - representative details/validation.spec.js:67]: Add test to validate that user can submit empty organisation

File: apps/fbis-ccf/translations/src/en/fields.json [hunk 4]
```diff
@@ -47,14 +47,24 @@
   "details": {
```
Comment (AlexFranklinHODDaT) [inline apps/fbis-ccf/translations/src/en/fields.json:47]: Should this be representative-details?

Comment (AlexFranklinHODDaT) [reply apps/fbis-ccf/translations/src/en/fields.json:47]: Also correct representative-details template

**Review by AlexFranklinHODDaT — CHANGES_REQUESTED** [review]

**Review by AlexFranklinHODDaT — APPROVED** [review]
